expense-splitter/src/components/context/SiteContext.jsx

```diff
@@ -1,60 +1,42 @@
-import React, {useState} from 'react'
+import React, { useState } from 'react'
 import { nanoid } from 'nanoid'
 import { createContext, useContext } from 'react'
+import db from '../../utils/localstoragedb'
 
 const SiteContext = createContext(null)
 
 export const UseDataContext = () => useContext(SiteContext)
 
-export const DataProvider = ({children}) => {
-    const initalGroup = [
-        {
-          name: 'Beach Lunch',
-          description: 'Saturday lunch by the beach',
-          budget: '85',
-          id: 't0unxnqoAwLdxsjOuxc5A',
-        },
-        {
-          name: 'Bar Night',
-          description: 'Night at the bar with co-workers',
-          budget: '300',
-          id: 'NoWgbblj8apsEHZ0IVMC4',
-        },
-        {
-          name: 'Urth Cafe Brunch',
-          description: 'Brunch with the baseball team',
-          budget: '120',
-          id: 'P_PNnrbjurvmasxRtXEov',
-        },
-      ]
+export const DataProvider = ({ children }) => {
+  // initialize data from localStorageDB
+  const data = db
```

Comment:
The data declaration is unnecessary. Simply importing db from the utility file will initialize it.

Reply:
That was just there for console logging. It's removed.